src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    UserService userService = new UserService();
```

Comment:
Could you add access modifier?

Reply:
I don't get it. To which object? Both user credentials and "users" array are private.